src/scala/io/bazel/rules_scala/scaladoc_support/BUILD

```diff
@@ -0,0 +1,17 @@
+load("//scala:scala.bzl", "scala_binary")
+
+# A simple scala_binary to run scaladoc.
+# `bazel run` this target with "-help" as a param for usage text:
+# bazel run -- "//src/scala/io/bazel/rules_scala/scaladoc_support:scaladoc_generator" -help
+scala_binary(
```

Comment:
I put this target in `src/scala/io/bazel/rules_scala/scaladoc_support` because I initially had an actual Scala program do heavy lifting, but it turned out to be unnecessary and I could pass in everything through scaladoc/scalac args.

Let me know if this should be moved to the `//scala` package or something.

Reply:
I think this is fine.

Reply:
Cool - happy to keep this here in case we do need to introduce a real Scala program.